src/Components/events/DetailEvent.js

```diff
@@ -20,26 +16,20 @@ const DetailEvent = () => {
   const eventId = useParams().id;
   const location = useLocation();
   const userId = location.state.userId;
+  const eventsList = location.state.eventsList;
   const [goToCreateNewEvent, setGoToCreateNewEvent] = useState(false);
   const [goToDashboard, setGoToDashboard] = useState(false);
   const [goToEditEvent, setGoToEditEvent] = useState(false);
   const [eventToEdit, setEventToEdit] = useState("");
-  const [eventsList, setEventsList] = [];
-
-  useEffect(() => {
-    async function getEvents() {
-      const eventsFound = await getEventsFromServer();
-      setEventsList(eventsFound.events);
```

Comment:
in here update the state calling `setEvent(eventFound)` and pass the event we got from calling `getEventFromServer()`, that way we will have the event available